requirements.txt

```diff
@@ -1,8 +1,12 @@
+versiontag
 mutmut==1.5.1
 pytest==5.0.0
 xlrd==1.2.0
 xlsxwriter==1.2.1
-pandas==0.24.2
+pandas==1.0.5
 codecov==2.0.15
 pytest-cov==2.7.1
-pylint==2.3.1
\ No newline at end of file
+pylint==2.3.1
+lizard==1.17.3
+vulture==1.3
+pyspelling==2.6
```

Comment:
All need not be part of the package requirement. Keep only the necessary package in the package requirement. rest could be seperate file inside build_script

Reply:
Corrected